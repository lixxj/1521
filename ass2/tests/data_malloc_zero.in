a = malloc 1
b = malloc 0