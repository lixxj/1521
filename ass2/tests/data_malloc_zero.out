test3: couldn't allocate 0 bytes for `b'
+00000 (F, 5000) 
[a] +00008 
+00000 (A,   12) +00012 (F, 4988) 
